tests/foreman/ui/test_organization.py

```diff
@@ -355,23 +355,121 @@ def test_negative_update(self):
 
     @run_only_on('sat')
     @tier2
-    def test_positive_remove_domain(self):
+    def test_positive_create_with_subnet(self):
+        """Create Org with subnet.
+
+        :id: 6736cd82-a2b0-4fc0-a2bc-99c9f13464d7
+
+        :assert: Org created with subnet.
+
+        :CaseLevel: Integration
+        """
+        with Session(self.browser) as session:
+            for subnet_name in generate_strings_list():
+                with self.subTest(subnet_name):
+                    # Create subnet using nailgun
+                    subnet = entities.Subnet(
+                        name=subnet_name,
+                        network=gen_ipaddr(ip3=True),
+                        mask='255.255.255.0'
+                    ).create()
+                    self.assertEqual(subnet.name, subnet_name)
+                    org_name = gen_string('alpha')
+                    make_org(session, org_name=org_name, subnets=[subnet_name])
+                    self.assertIsNotNone(self.org.search(org_name))
+                    # self.org.update(org_name, new_subnets=[subnet_name])
+                    self.org.search_and_click(org_name)
+                    session.nav.click(tab_locators['context.tab_subnets'])
+                    element = session.nav.wait_until_element(
+                        common_locators['entity_deselect'] % subnet_name)
+                    self.assertIsNotNone(element)
+
+    @run_only_on('sat')
+    @tier2
+    def test_positive_update_subnet(self):
+        """Add and Remove subnet by using organization name and subnet name.
+
+        :id: bc59bdeb-b538-4473-a096-e4de2454497d
+
+        :assert: subnet is added then removed.
+
+        :CaseLevel: Integration
+        """
+        with Session(self.browser) as session:
+            org_name = session.get_org_name()
+            for subnet_name in generate_strings_list():
+                with self.subTest(subnet_name):
+                    # Create subnet using nailgun
+                    subnet = entities.Subnet(
+                        name=subnet_name,
+                        network=gen_ipaddr(ip3=True),
+                        mask='255.255.255.0',
+                    ).create()
+                    self.assertEqual(subnet.name, subnet_name)
+                    self.org.update(
+                        org_name, subnets=[subnet_name], select=True)
+                    self.org.search_and_click(org_name)
+                    session.nav.click(tab_locators['context.tab_subnets'])
+                    element = session.nav.wait_until_element(
+                        common_locators['entity_deselect'] % subnet_name)
+                    # Item is listed in 'Selected Items' list and not
+                    # 'All Items' list.
+                    self.assertIsNotNone(element)
+                    self.org.update(org_name, subnets=[subnet_name])
```

Comment:
@rochacbruno On update tests always add and remove entities related to Org. And once each test only check the property under test, one failing test will not impact another. But I can be sure for the tests unders this module, it could maybe impact others that start to use it. So, like you, I am going to wait for @oshtaier's opinion ;)